package.json

```diff
@@ -18,9 +18,18 @@
   },
   "homepage": "https://github.com/skateray17/ExadelFreeFoodDeliveryTask#readme",
   "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.6.1",
+    "babel-preset-es2015": "^6.24.1",
     "eslint": "^4.9.0",
     "eslint-config-airbnb-base": "^12.1.0",
     "eslint-plugin-import": "^2.7.0",
+    "handlebars": "^4.0.11",
```

Comment:
Remove handlebars and dist/handlebars.js dependency from devDependencies.

Install handlebars with:
npm install --save handlebars

Note:
Libraries that you want to reference with import X from 'X' should be installed as regular dependencies not devDependencies.